reflect/reflect_test.go

```diff
@@ -136,6 +137,99 @@ func TestMapToStruct(t *testing.T) {
 	assert()
 }
 
+func TestStructFieldMap(t *testing.T) {
+	m, ok := StructFieldMap(Tweet{}, false)
+	if !ok {
+		t.Fatalf("expected field map to be created")
+	}
+
+	if timeline, ok := m["Timeline"]; ok {
+		if timeline.Name() != "Timeline" {
+			t.Errorf("Timeline should have name 'Timeline' but got %s", timeline.Name())
+		}
```

Comment:
I wonder if it's worth introducing an assertion library to make these less verbose (e.g. `testify/assert`) but appreciate we might not necessarily want to add a new dependency/way of testing things

Reply:
In fact I've seen that we're using `testify/assert` below in `scanner_test.go`, so we should almost definitely update these tests to be consistent

Reply:
I didn't want to drastically change this file and thus have decided to stick with the existing convention (especially because this is a different package too within gocassa). I don't think this needs to be a blocker